streams/shared/src/main/scala/zio/stream/ZSink.scala

```diff
@@ -1597,6 +1597,16 @@ object ZSink extends ZSinkPlatformSpecificConstructors {
   )(implicit trace: Trace): ZSink[R, E, In, L, Z] =
     new ZSink(ZChannel.unwrap(zio.map(_.channel)))
 
+  /**
+   * A generalized version of `timed`.
+   */
+  def summarized[R, E, B, C](
```

Comment:
@adamgfraser I don't see this change in current `series/2.x`. Is that intentional?

Reply:
@sideeffffect The changes to `summarized` were supposed to be a back port of the existing functionality from ZIO 2.0 to ZIO 1.0 so I left the API as is though we could certainly add later.

Reply:
Ah, so it's a bit more complicated :smile: Fair enough, I just wanted to make sure nothing slipped though the cracks, so to speak :wink:

Reply:
Definitely!